packages/terra-action-footer/src/ActionFooterContainer.scss

```diff
@@ -0,0 +1,21 @@
+$default-padding: 0.357142857142857rem;
+
+:local {
+  .action-footer-container {
+    background-color: var(--terra-action-footer-background-color, transparent);
+    border-top: var(--terra-action-footer-border-top, 0.071428571428571rem solid #cfd0d1);
+    display: flex;
+  }
+
+  .has-children {
+    min-height: 2.857142857142857rem;
+    padding-bottom: var(--terra-action-footer-padding-bottom, $default-padding);
+    padding-left: var(--terra-action-footer-padding-left, $default-padding);
+    padding-right: var(--terra-action-footer-padding-right, $default-padding);
+    padding-top: var(--terra-action-footer-padding-top, $default-padding);
+  }
+
+  .is-empty {
+    height: 0.714285714285714rem;
```

Comment:
nit: Can we change this to be `0.7142857143rem`. It's still 10px, it just better aligns with the called out naming convention equivalents [here](http://engineering.cerner.com/terra-core/#/site/spacer).

Reply:
same goes w/ the `default-padding`

Reply:
👍consistency with spacer sounds good